BitwardenShared/UI/Platform/Application/Views/BitwardenTextField.swift

```diff
@@ -162,11 +197,27 @@ struct BitwardenTextField<TrailingContent: View>: View {
         self.footer = footer
         self.canViewPassword = canViewPassword
         self.passwordVisibilityAccessibilityId = passwordVisibilityAccessibilityId
-        self.placeholder = placeholder ?? ""
         _text = text
         self.title = title
         self.trailingContent = trailingContent()
     }
+
+    // MARK: Private
+
+    /// The title/placeholder text for the field.
+    @ViewBuilder
+    private func titleText(showPlaceholder: Bool) -> some View {
+        if let title {
+            Text(title)
+                .styleGuide(
+                    showPlaceholder ? .body : .subheadline,
+                    weight: showPlaceholder ? .regular : .semibold, // semibold ??
```

Comment:
🤔 Is the comment `// semibold ??` resolved or shall we ask product about this?

Reply:
Ah, good catch, that's been resolved.